disk_expand/gce-disk-expand-el7.spec

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 Name: gce-disk-expand
 Summary: Google Compute Engine root disk expansion utilities for EL7
-Version: 1.0.1
+Version: 1.0.2
```

Comment:
Please move this into a follow-up. The development branch contains several packages needing version bump.